cards/card_component.js

```diff
@@ -80,6 +113,73 @@ BaseCard["{{componentName}}"] = class extends ANSWERS.Component {
     }
   }
 
+  /**
+   * Triggers the event passed in dispatched from the given selector
+   * @param {string} selector selector to dispatch event from
+   * @param {string} event event to fire
+   * @param {Object} settings additional settings
+   */
+   _triggerCustomEvent(selector, event, settings) {
+    let e = this._customEvent(event, settings);
+    this._container.querySelector(selector).dispatchEvent(e);
+  }
+
+  /**
+   * _customEvent is an event constructor polyfill
+   * @param {string} event event to fire
+   * @param {Object} settings additional settings
+   */
+  _customEvent(event, settings) {
```

Comment:
I think this is duplicated with the base direct answer card, is there a common place we can put this?